src/components/BulkUploadListItem.tsx

```diff
@@ -90,7 +89,7 @@ export const BulkUploadListItem = ({ upload }: BulkUploadListItemProps) => {
 						{datum}
 					</>
 				),
-			null,
+			'',
```

Comment:
This was changed from null to empty string because typescript was barfing at the assignment of a string to a null value. I am unsure why this wasn't causing an error before, but my understanding is that this should not make a difference as far as output is concerned? I.e if the string was reduced to null before it would be output as an empty string either way

Reply:
I don't think that's quite accurate; look at the `null` check in the reducer function:

https://github.com/PhilanthropyDataCommons/front-end/blob/35cb42479ba077cd0580dc402245490030c7c717/src/components/BulkUploadListItem.tsx#L80-L94

I this will cause the first item to have a bullet in front of it, where previously it didn't:

![before](https://github.com/user-attachments/assets/17e0dfd7-005b-4ef8-9644-776387340142)
![after](https://github.com/user-attachments/assets/c6c1e743-4e2b-4d2b-9eb9-06d571827eda)

The prod instance doesn't have any bulk uploads, so this change can't be tested against that - you'll need to make sure your local instance has some and point it at that.

I think this can best be solved by removing the initial value entirely, as well as the trigraph operator; as [the MDN docs on `reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) say,

> The first time that the callback is run there is no "return value of the previous calculation". If [no initial value is supplied,] the array element at index 0 is used as the initial value and iteration starts from the next element (index 1 instead of index 0).

This should also be extracted into a separate commit.

Reply:
This is closed by #804, which should be merged in before this PR